src/interfaces/store-interfaces/state-interfaces/name-request-interface.ts

```diff
@@ -7,6 +7,14 @@ export interface NameRequestIF {
   filingId: number | null
 }
 
+// Account information interface
```

Comment:
Please move this to a new file. THIS file contains NR-specific IFs.

Reply:
oops... fixed